2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,45 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+let recipeCard1 = {
+  title: "Ramen",
+  serves: 1,
+  ingredients: ["noodles", "onions", "soySauce", "chickenBroth", "egg"],
+}
+let recipeCard2 = {
+  title: "Cheese cake",
+  serves: 5,
+  ingredients: ["digestive biscuit", "soft cheese", "double cream", "strawberry"],
+}
+
+let recipeCard3 = {
+  title: "ham sandwich",
+  serves: 2,
+  ingredients: ["bread", "ham", "butter"],
+}
+
+let recipeCard4 = {
+  title: "smoothie",
+  serves: 2,
+  ingredients: ["banana", "strawberry", "yoghurt"],
+}
+
+let recipeCard5 = {
+  title: "Mole",
+  serves: 2,
+  ingredients: ["cinnamon", "cumin", "cocoa"],
+}
+
+function displayRecipes (recipeCard){
+  console.log(recipeCard.title);
+  console.log("serves: " + recipeCard.serves); 
+  console.log("Ingredients: ")// this shows ingredients
+  console.log(recipeCard.ingredients.join('\r\n'));
+}
+
+console.log(displayRecipes(recipeCard1));
```

Comment:
The console.log on line 62 and below, log undefined. Any ideas why this might be the case?

Reply:
updated. thanks Maten.